contributing.md

```diff
@@ -61,46 +152,133 @@ lines, etc.), ``isort`` for automatic import sorting, and ``flake8`` for linting
 
-Documentation is written using RST, and is located both inline and within the /sphinx directory.
-Any changes to the analysis methodology should be discussed there or offline. Once a methodology change is decided,
-create new tickets in this repository towards implementing the change.
+```bash
+# Assuming you're already in OpenOA/
+cd sphinx/
+
+# Build the docs
+make html
+```
 
 ## Testing
 
-All code should be paired with a corresponding unit or integration test.
-OpenOA uses pytest and the built in unittest framework.
-For instructions on running tests, please see the [Readme](https://github.com/NREL/OpenOA/tree/develop#Testing).
+All code should be paired with a corresponding unit or integration test. OpenOA uses both pytest and
+the built in unittest framework.
+
+To run the tests you can use any of the following commands, depending on your needs.
+
+1. All the tests and check for test coverage:
+
+   ```bash
+   pytest --cov=openoa
+   ```
+
+2. All the tests:
+
+   ```bash
+   pytest
+   ```
+
+3. Only the unit tests:
+
+   ```bash
+   pytest test/unit
+   ```
+
+4. Only the regression (integration) tests:
+
+   ```bash
+   pytest test/regression
+   ```
+
+## Pull Request
+
+Pull requests must be made for all changes. Most pull requests should be made against the develop
+branch unless patching a bug that needs to be addressed immediately, and only core developers should
+make pull requests to the main branch.
+
+All pull requests, regardless of the base branch, must include updated documentation and pass all
+unit tests and integration tests. In addition, code coverage should not be negatively affected.
+
+### Scope
+
+Encapsulate the changes of one issue, or multiple if they are highly related. Three small pull
+requests is greatly preferred over one large pull request. Not only will the review process be
+shorter, but the review will be more focused and of higher quality, benefitting the author and code
+base. Be sure to write a complete description of these changes in the pull request body.
+
+### Tests
+
+All tests must pass. Pull requests will be rejected or have changes requested if tests do not pass,
+or cannot pass with changes. Tests are automatically run through Github Actions for any pull request
+or push to the main or develop branches, but should also be run locally before submission.
+#### Test Coverage
+
+The testing framework described below will generate a coverage report from the tests run through
+GitHub Actions. Please ensure that your work is fully covered by running them locally with the
+coverage report enabled.
+
+### Documentation
+
+Include any relevant changes to inline documentation, docstrings, and any of the RST files located
+in `OpenOA/sphinx/`. Pull requests will not be accepted until these changes are complete.
+
+### Changelog
+
+All changes must be documented appropriately in CHANGELOG.md in the [Unreleased] section.
 
 ## Release Process
 
```

Comment:
May be worth adding a couple sentences clarifying this section is for maintainers of the NREL/OpenOA repo only, and does not apply to your own fork.